pkg/controllers/podgroup_controller.go

```diff
@@ -102,29 +102,28 @@ func (r *PodGroupReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 	pods := podList.Items
 
 	pgCopy := pg.DeepCopy()
-	switch pgCopy.Status.Phase {
-	case "":
+
+	if pgCopy.Status.Phase == "" {
```

Comment:
why change this `switch` into `if else` ?

Reply:
I changed the `switch-case` statements to` if-else` statements because of a reason below. 

The `switch-case` logic uses the past value of `pgCopy.Status.Phase` even if the `pgCopy.Status.Phase` changes within `switch-case` statements. As a result, we can't pass this `switch-case` statements at once, and that's why I preferred `if-else` statements. 

But this was not a bug, but a matter of preference, so I reverted to the original code. 
Thanks for your comments.